apps/web/src/pages/Pools/IndexPoolPage/components/PoolTable.tsx

```diff
@@ -180,20 +215,41 @@ export const PoolTable = ({ pools }: PoolTableProps) => {
           description:
             "Title displayed for the Pool table Group and Level column.",
         }),
-        accessor: ({ classifications }) =>
-          classifications?.map((classification) => {
-            return (
-              <Pill
-                key={`${classification?.group}-${classification?.level}`}
-                color="primary"
-                mode="outline"
-              >
-                {classification?.group}&#8209;{classification?.level}
-              </Pill>
-            );
-          }),
+        accessor: (d) => {
+          let classificationsString = "";
+          if (d.classifications && d.classifications.length > 0) {
+            d.classifications.forEach((classification) => {
+              if (classification) {
+                const groupLevelString = `${classification.group}-${classification.level} `;
+                classificationsString += groupLevelString;
+              }
+            });
+          }
+          return classificationsString;
+        },
+        Cell: ({ row }: IndividualCell) => {
+          const filteredClassifications = row.original.classifications
+            ? row.original.classifications.filter(
+                (classification) => !!classification,
+              )
+            : null;
+          const pillsArray = filteredClassifications
+            ? filteredClassifications.map((classification) => {
+                return (
+                  <Pill
+                    key={`${classification?.group}-${classification?.level}`}
+                    color="primary"
+                    mode="outline"
+                  >
+                    {classification?.group}&#8209;{classification?.level}
+                  </Pill>
+                );
+              })
+            : null;
+          return classificationsCell(pillsArray);
+        },
```

Comment:
Could this all be moved to the `classificationsCell` function?

Reply:
done in https://github.com/GCTC-NTGC/gc-digital-talent/pull/5631/commits/42fe014afa9588f87a12b3880dce81c6fd9c00f2